ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -197,27 +200,6 @@ def _arrays_idx_n_dtypes(draw):
     return xs, input_dtypes, unique_idx
 
 
-@st.composite
-def _chunk_helper(draw):
-    dtype, x, shape = draw(
-        helpers.dtype_and_values(
-            available_dtypes=helpers.get_dtypes("float"),
-            min_num_dims=1,
-            ret_shape=True,
-        )
-    )
-    axis = draw(helpers.get_axis(shape=shape, force_int=True))
-    if shape[axis] == 0:
-        chunks = 0
-    else:
-        factors = []
-        for i in range(1, shape[axis] + 1):
-            if shape[axis] % i == 0:
```

Comment:
May be you've rebased incorrectly, these changes shouldn't be shown here. Can you please re-check this?

Reply:
Hi @bipinKrishnan ..I actually moved this testing function since `torch`  and `paddle`were using the same function and placed the common function into the helper directory.I can rebase and correct it now.No problem

Reply:
Yeah since this PR is not related to torch, its better to keep the files untouched. Hope that makes sense!

Reply:
Hi @bipinKrishnan could you take another look ? I have removed the changes that were related to `torch`.